### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by qntnt**
Adding some README info about getting started with project contributions.

### CODE REVIEW

**Review by The-Tasty-Jaffa — CHANGES_REQUESTED** [review]
These steps don't entirely follow how Red bot is deployed.
This is particularly true for step 5

Linking [this](https://docs.discord.red/en/stable/getting_started.html) Could be a good idea too.
This is certainly a great start!

**Comment by The-Tasty-Jaffa** [comment]
This looks good.
We could direct people towards https://docs.discord.red/en/stable/install_linux_mac.html
and https://docs.discord.red/en/stable/install_windows.html respectively too.
Since it has all the updated stuff, reasonably comprehensive and simple to follow.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+6. Assign yourself to a task on our [project board](https://github.com/PortZeroGroup/PZ-DiscordBot/projects/1) or [issue tracker](https://github.com/PortZeroGroup/PZ-DiscordBot/issues)
+7. Code and test locally. (**Do not check in your credentials**)
+8. Submit a Pull Request from your fork back into [PZ-DiscordBot](https://github.com/PortZeroGroup/PZ-DiscordBot). (**Do not check in your credentials**)
+
+## Deployment
+
+**Summary**
+We will automatically deploy release branches ([TODO](https://github.com/PortZeroGroup/PZ-DiscordBot/issues/8)) when commits are made. To do this safely, changes will merge into a "staging" branch ([TODO](https://github.com/PortZeroGroup/PZ-DiscordBot/issues/7)) which is then immediately deployed to our testing environment. After some time, when automated or manual testing is done, the testing branch will merge into the release branch and the production bot will automatically redeploy.
```
Comment (The-Tasty-Jaffa) [inline README.md]: Not Sure about this bit.
None of this is yet implemented, it might be worth removing this section for now.

**Review by The-Tasty-Jaffa — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
 The Public repo for the PZ discord bot
+
+## Getting Started
+
+This bot is built with Python, using the Red framework. Here are the docs for [Red](https://docs.discord.red/en/stable/index.html).
+
+At a high level, bot commands and features are implemented as plugins (Red calls them "cogs" so we'll call them that from here on).
+
+To get started contributing to PZ Bot:
+1. Join the Port Zero discord server and join the project (#discord-bot-dev channel)
```
Comment (daswanderer) [inline README.md]: Nit: Can we make this a link to the PZ discord

**Review by daswanderer — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MrLotU) [inline README.md]:
```suggestion
1. Join the [Port Zero discord server](https://discord.gg/jmHWZA4vg5) and join the project (#discord-bot-dev channel)
```

**Pull request merged**